src/sentry/incidents/models.py

```diff
@@ -416,6 +416,20 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_alertruletriggeraction"
 
+    @property
+    def human_desc(self):
```

Comment:
Since this is all display logic, it might make sense to actually move it into the serializer.